### PR TITLE
[React - Vue] - Migrate E2E Tests from Cypress to Playwright 

### PR DESCRIPTION
**Pull request by noriega2112**
## What this does

Switches the E2E testing libraries from Cypress to Playwright for the React and Vue sides

## Checklist
- [x] Switch to playwright on React Side
- [x] Switch to playwright on Vue Side
- [x] Switch E2E GH workflow

## How to test

Add user steps to achieve desired functionality for this feature.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+const PASSWORD = 'PASSWORD'
+
+test('Login workflow', async ({ page }) => {
+  await page.goto('/sign-up')
+  await page.getByTestId('first-name').fill('playwright')
+  await page.getByTestId('last-name').fill('e2e test')
+  await page
+    .getByTestId('email')
+    .fill(`playwright${Math.floor(Math.random() * 1000)}@thinknimble.com`)
```
Comment (whusterj) [inline {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts:11]: I suggest using a UUID or a larger number here to greatly reduce the chance of a collision.

**Review by whusterj — COMMENTED** [review]
The code looks good, but the Chrome and Firefox e2e tests have not passed yet:

image

Once these are passing and the conflict with `main` is resolved, this should be good to go!

File: {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts (hunk 1 above)
Comment (noriega2112) [reply {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts]: yeah you are right, I replaced it for a timestamp instead as its value is unique

**Comment by noriega2112** [comment]
> The code looks good, but the Chrome and Firefox e2e tests have not passed yet:
> 
> image
> Once these are passing and the conflict with `main` is resolved, this should be good to go!

@whusterj  These checks shouldn't be showing as they were part of the `cypress.yml` github action that was before, I think they are still showing as they were required prior to opening the PR. 

Now this is being handled on the playwright config file which makes the tests to run on the 3 major browsers.

**Comment by whusterj** [comment]
> @whusterj These checks shouldn't be showing as they were part of the `cypress.yml` github action that was before, I think they are still showing as they were required prior to opening the PR.
> 
> Now this is being handled on the playwright config file which makes the tests to run on the 3 major browsers.

I see - that makes sense! They should disappear after we merge.

File: {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts (hunk 1 above)
Comment (whusterj) [reply {{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/sign-up.spec.ts]: There is still a chance of collision when browser tests are run in parallel, but that should be very unlikely, so this is a lot better. Thanks!

**Review by whusterj — APPROVED** [review]
Looks good to go!

**Pull request merged**